journali.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+echo "      _                              _ _ 
+     | |                            | (_)
+     | | ___  _   _ _ __ _ __   __ _| |_ 
+ _   | |/ _ \| | | | '__| '_ \ / _\` | | |
+| |__| | (_) | |_| | |  | | | | (_| | | |
+ \____/ \___/ \__,_|_|  |_| |_|\__,_|_|_|
+"
+
+BASE_COMMAND="docker-compose -f docker-compose.builder.yml run --rm"
+
+case $1 in
+    init)
+        echo " - Initializing repository"
+        git clone git@github.com:WesleyKlop/journali-frontend.git frontend
+        git clone git@github.com:WesleyKlop/journali-api.git api
+        ;;
+    install)
+        echo " - Installing frontend dependencies for the frontend"
+        eval $BASE_COMMAND install
+        ;;
+    migrate)
+        echo " - Migrating database"
+        # TODO
+        ;;
+    start)
+        echo " - Starting development environment"
+        docker-compose up -d
+        ;;
+    stop)
+        echo " - Stopping development environment"
+        docker-compose down
+        ;;
+    yarn | cargo)
+        echo "Running: "$BASE_COMMAND ${@:1}
+        eval $BASE_COMMAND ${@:1}
+        ;;
+    help)
+        echo "Available commands:
+ - init: Initialize the child repositories
+ - install: Intall frontend node dependencies
+ - migrate: Migrate the database
+ - start: Start the development environment
+ - stop: Stop the development environment
+ - yarn [command]: Run any yarn command on the frontend environment
+ - cargo [command]: Run any cargo command on the api environment
+ - help: This command"
+        ;;
+    *)
+        echo "Unknown command \"$1\""
```

Comment:
`echo " - Unknown command"` to match all other echo's

Reply:
Nah because it's an error message I don't think we should do that

Reply:
I changed it to send things to stderr does that help